src/components/track.tsx

```diff
@@ -3,49 +3,84 @@ import * as React from "react";
 import { accentColor } from "../constants";
 import { usePublicFileURL } from "../hooks/use-public-file-url";
 
+const TitleContainer = styled.div`
+  position: absolute;
+
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4.5rem;
+  overflow: hidden;
+  transition: 0.5s ease;
+
+  background-color: rgba(0, 0, 0, 0.75);
+
+  -webkit-backdrop-filter: blur(10px);
+  border-top-right-radius: 3px;
+  border-bottom-right-radius: 3px;
+
+  z-index: 0;
+  cursor: default;
+`;
+
 const ImageContainer = styled.div`
   position: relative;
 
   display: block;
-  width: 100%;
   height: 15rem;
   overflow: hidden;
   margin-bottom: 1rem;
 
   border-radius: 5px;
 
   background-size: 100% auto;
-  background-position: center center;
   background-color: white;
+  background-position: center center;
+
+  &:hover ${TitleContainer} {
+    height: 100%;
```

Comment:
I'm wondering whether fading to a darker background would be helpful, too, e.g. to `background-color: rgba(0, 0, 0, 0.9)` or similar

Reply:
Are you talking about the background color for TitleContainer or ImageContainer? Line Numbers make it hard to discern what you're talking about

Reply:
I meant the background of the container holding the title content, `TitleContainer`, which has `background-color: rgba(0, 0, 0, 0.75)`